subsys/bluetooth/controller/Kconfig

```diff
@@ -38,7 +38,7 @@ config BT_LL_SOFTDEVICE
 	select BT_HAS_HCI_VS
 	select BT_CTLR_DF_SUPPORT if HAS_HW_NRF_RADIO_DFE
 	select BT_CTLR_LE_POWER_CONTROL_SUPPORT
-	depends on (SOC_SERIES_BSIM_NRFXX || SOC_SERIES_NRF52X || SOC_NRF5340_CPUNET)
+	depends on (SOC_COMPATIBLE_NRF52X || SOC_COMPATIBLE_NRF5340_CPUNET)
```

Comment:
@jori-nordic , do we still need this, or should we depend on that the dragoon west group is enabled?

Reply:
will be determined today. It depends if we want to keep building with the prebuilt lib or only allow from source.

Reply:
we don't need it anymore, I'll post a PR to remove it